falcon/vendor/mimeparse/mimeparse.py

```diff
@@ -98,7 +98,7 @@ def quality_and_fitness_parsed(mime_type, parsed_ranges):
         if type_match and subtype_match:
 
             # 100 points if the type matches w/o a wildcard
-            fitness = type == target_type and 100 or 0
+            fitness = 100 if type == target_type else 0
```

Comment:
Let us revert these changes. Sensible they may be, we have vendored this module from https://github.com/dbtsai/python-mimeparse, and do not want to make any modifications unless absolutely necessary.